packages/radix-vue/src/Select/SelectContentImpl.vue

```diff
@@ -184,9 +188,9 @@ function handleKeyDown(event: KeyboardEvent) {
   }
 }
 
-const pickedProps = computed(() => {
+const popperContentProps = computed(() => {
   if (props.position === 'popper')
-    return props
+    return useForwardProps(props)
```

Comment:
`useForwardProps` should be caled within the setup context, as it is using `getCurrentInstance` internally. Let's move it out from `computed` ya